src/libraries/FCAL/DFCALShower_factory.cc

```diff
@@ -203,6 +207,21 @@ jerror_t DFCALShower_factory::brun(JEventLoop *loop, int32_t runnumber)
 	}
       }
     }
+    loop->GetCalib("FCAL/nonlinear", nonlinear);
```

Comment:
I am not crazy about the name of this table.  Can it be more descriptive, like "FCAL/nonlinear_correction", or something like that?